environs/interface.go

```diff
@@ -664,3 +665,9 @@ type CloudEndpointChecker interface {
 	// endpoint and returns nil if no problems.
 	ValidateCloudEndpoint(ctx envcontext.ProviderCallContext) error
 }
+
+// DesiredReplicasGetter defines a method for getting the desired
+// replicas for a given application.
+type DesiredReplicasGetter interface {
+	DesiredReplicas(name string, deploymentType k8s.K8sDeploymentType) (int, error)
```

Comment:
```suggestion
	DesiredReplicas(name string, deploymentType deploymenttype.DeploymentType) (int, error)
```

Reply:
This changed with the naming suggested by @manadart .